NetKAN/Arkas-Config-Med-Clouds.netkan

```diff
@@ -0,0 +1,21 @@
+{
+	"spec_version"	:	1,
+	"identifier"	:	"Arkas-Config-Med-Clouds",
+	"name"		:	"Arkas Med Cloud configuration",
```

Comment:
name does not need the word "configuration". `"Arkas - Medium Clouds"` would be sufficient.